modin/pandas/test/test_general.py

```diff
@@ -801,3 +802,9 @@ def test_empty_dataframe():
 def test_empty_series():
     s = pd.Series([])
     pd.to_numeric(s)
+
+
+def test_to_timedelta():
+    # This test case comes from
+    # https://github.com/modin-project/modin/issues/4966
+    eval_general(pd, pandas, lambda lib: lib.to_timedelta(lib.Series([1])))
```

Comment:
We should test float and int data, at least, with enough data to fill a few partitions. `test_data` in the utils has exactly those types. We can adapt the code here to parametrize by the data type-- maybe we don't need a method as complicated as `create_test_series`. https://github.com/modin-project/modin/blob/0a2c0de4451f7e2e8f337a9478d7595473aa348e/modin/pandas/test/test_series.py#L210-L212